tests/testdata/plugins/test-plugin-with-errors/test-plugin-with-errors.php

```diff
@@ -16,3 +16,4 @@
  */
 
 include_once 'i18-usage-error-file.php';
+include_once 'enqueued-scripts-size-check.php';
```

Comment:
Rather than creating a single test plugin that has a bunch of errors, we should create _individual_ test plugins, one for each kind of test; otherwise things will get messy very quickly with our test code as the single plugin will do a bunch of things over time.

So let's not add to the existing test plugin but rather add a new one like `test-plugin-with-enqueued-scripts-size-errors`. We may also want to rename the existing `test-plugin-with-errors` to `test-plugin-with-i18n-usage-errors`.

Reply:
Thanks @felixarntz I've updated all the test plugins in the PR to follow this new structure.